.azure/templates/composer-install.yml

```diff
@@ -1,3 +1,12 @@
-steps:  
+steps: 
+- task: Cache@2
+  inputs:
+    key: 'composer | "$(Agent.OS)" | composer.lock'
```

Comment:
Unless `composer` refers to a file, you need to wrap it in double quotes.

Reply:
I believe as long as the string doesn't contain anything that ["looks like a file path"](https://dev.azure.com/graycore/open-source/_build/results?buildId=5540&view=logs&j=c4ddea8a-7257-56f5-e738-ae9be1d6d628&t=07af7bd4-c48a-58b2-f5a2-4731872cd0fe) the string "composer" will be okay.  I looked into this because I thought that was an interesting nuance.

Reply:
Well I guess as long as azure doesn't complain. When I was writing the incremental build ci, it was interpreting literally everything that wasn't in quotes as a filepath

Reply:
TBH -- I took this directly from https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#phpcomposer

If it's wrong, blame MSFT for bad docs.